libretroshare/src/serialiser/rstypeserializer.h

```diff
@@ -910,13 +945,190 @@ struct RsTypeSerializer
 	                       t_RsTlvList<TLV_CLASS,TLV_TYPE>& member,
 	                       RsJson& jDoc );
 
-	[[noreturn]] static void fatalUnknownSerialJob(int j)
+//============================================================================//
```

Comment:
There was some bullet proof code in FileListIO named Read125Size and Write125Size. I remember fixing nasty bugs in it, so we need to be careful using some new code here. It would be safer to copy the code in FileListIO, put it here, and call it from FileListIO so we have one instance of that code only.

Reply:
I read that code, the idea is similar but this implementation is more generic, the 125 version can handle integer up to uint32, while this can handle any integer size, also it is efficient handling signed integers

Reply:
ok. Because 125 code is used in crypto parts too, I would not replace it by this new code that is too high level, and keep your new code where it is.